src/main/java/duke/task/DukeList.java

```diff
@@ -0,0 +1,135 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+/**
+ * Represents a list of tasks.
+ * Provides methods to add, delete, and manipulate tasks within the list.
+ */
+public class DukeList {
+    ArrayList<Task> arr;
```

Comment:
Maybe the variable name could be more descriptive, like 'taskList' or 'dukeList'